src/main/java/screens/Window.java

```diff
@@ -60,6 +62,11 @@
     private static InfoScreen infoScreen = null;
 
     /**
+     * The load bar which shows how far the parser is.
+     */
+    private static ProgressBar pB;
+    
+    /**
```

Comment:
As I mentioned, checkstyle does not give me an error and I can not find the trailing space myself either.